assets/src/components/propertiesPanel/vehiclePropertiesPanel.tsx

```diff
@@ -27,7 +27,9 @@ import { fullStoryEvent } from "../../helpers/fullStory"
 
 interface Props {
   selectedVehicle: Vehicle
-  initialTab?: TabMode
+  tabMode: TabMode
+  setTabMode: React.Dispatch<React.SetStateAction<TabMode>>
```

Comment:
suggestion: maybe rename these to `onChangeTab`?

Reply:
I went with `onChangeTabMode` (I wanted to avoid other senses of the word "tab," though I guess "tab mode" isn't inherently very descriptive). 7f6ce4dd